public/app/plugins/datasource/grafana-azure-monitor-datasource/query_ctrl.ts

```diff
@@ -581,13 +581,9 @@ export class AzureMonitorQueryCtrl extends QueryCtrl {
   }
 
   /* Application Insights Section */
-
   getAppInsightsAutoInterval() {
-    const interval = this.templateSrv.getBuiltInIntervalValue();
-    if (interval[interval.length - 1] === 's') {
-      return '1m';
-    }
-    return interval;
+    // TODO no way to get this currently from angular
```

Comment:
can we remove the whole function?  lots has changed in azure now that it is mostly a backend plugin, but the frontend still has many function around

Reply:
Think it's still used form UI, but this functionality has been broken since v6

Reply:
But can see if I can update the angular template that calls this